packages/office-ui-fabric-react/src/components/Callout/examples/Callout.FocusTrap.Example.tsx

```diff
@@ -95,14 +95,14 @@ export class CalloutFocusTrapExample extends React.Component<{}, ICalloutFocusTr
               <div className={styles.inner}>
                 <div>
                   <p className={styles.subtext}>
-                    Message body is optional. If help documentation is available, consider adding a link to learn more at the bottom.
+                    Content is wrapped in a Focus Trap Zone so that user cannot accidently tab out of this callout.
```

Comment:
Probably either use the literal control name or lowercase the words
```suggestion
                    Content is wrapped in a FocusTrapZone so that user cannot accidently tab out of this callout.
```